azure-pipelines.yml

```diff
@@ -26,7 +32,7 @@ steps:
     volumes: |
       $(build.sourcesDirectory):/srv/jekyll
       $(build.binariesDirectory):/srv/jekyll/_site
-    containerCommand: 'sh -x -c "bundle install && JEKYLL_ENV=production jekyll build"'
+    containerCommand: 'sh -x -c "ls -l /srv/jekyll && chmod a+w Gemfile.lock && bundle install && JEKYLL_ENV=production jekyll build"'
```

Comment:
@zemian I am not 100% sure why the file isn't writable, but this workaround (`chmod`) seems to work...  The only part that's weird is that it has not been needed for ehcache, but that's possibly because nobody has made the same changes there to Gemfile.lock, or, possibly, because Gemfile there has different deps that affect execution somehow.